## PS 3 Group Project 2  - Armed Conflicts

In this project you will be working with on major armed conflicts from the <a href="https://ucdp.uu.se/">Uppsala Conflict Data Program</a>. The following block of code loads it up as `conflicts`, and also imports some libraries you may want to use.

In [1]:
import pandas as pd
import numpy as np
import seaborn as sns 
from scipy import stats
import matplotlib.pyplot as plt
from matplotlib import pyplot as plt
import statsmodels.formula.api as smf

conflicts = pd.read_csv("data/cleaned/conflict_cleaned.csv")
conflicts

Side A  Conflict ID  \
0       Afghanistan          333   
1       Afghanistan          431   
2       Afghanistan        13637   
3       Afghanistan        13692   
4           Algeria          278   
..              ...          ...   
187  United Kingdom          273   
188  United Kingdom          286   
189  United Kingdom          315   
190         Uruguay          319   
191      Uzbekistan          415   

                                                Side B  Incompatibility  \
0    Harakat-i Inqilab-i Islami-yi Afghanistan , Hi...                2   
1                                               Russia                2   
2                                                   IS                1   
3             United Kingdom, United States of America                2   
4                                              Morocco                1   
..                                                 ...              ...   
187                   North Kalimantan Liberation Army                1   
188                                              FLOSY                1   
189                                               PIRA                1   
190                                      MLN/Tupamaros                2   
191                                                JIG                2   

     Year Observed  Intensity Level  Type of Conflict  Start Date  Ended  \
0             1982                2                 4  1975-07-02      0   
1             1979                1                 2  1979-12-27      1   
2             2016                2                 4  2015-02-09      0   
3             2001                2                 2  2001-10-07      1   
4             1963                1                 2  1963-10-08      1   
..             ...              ...               ...         ...    ...   
187           1962                1                 1  1962-12-08      1   
188           1964                1                 1  1964-11-25      0   
189           1985                1                 3  1970-08-11      0   
190           1972                1                 3  1970-04-13      1   
191           2004                1                 3  1999-02-16      1   

       End Date           GDP  Population  Military Expenditure % GDP  \
0           NaN  1.910135e+10  12882528.0                    1.173908   
1    1979-12-28  3.697940e+09  13411056.0                         NaN   
2           NaN  1.936264e+10  35383128.0                    0.956772   
3    2001-11-13           NaN  21606988.0                         NaN   
4    1963-11-04  2.703015e+09  11912803.0                    2.511485   
..          ...           ...         ...                         ...   
187  1962-12-31  8.124756e+10  53250000.0                    6.210394   
188  1967-11-29  9.440756e+10  54000000.0                    5.899068   
189  1991-11-26  4.892850e+11  56550268.0                    4.758479   
190  1972-12-31  2.189418e+09   2821461.0                    2.198068   
191  2004-07-30  1.203002e+10  25864350.0                    1.151780   

     Gini Index  Adult Literacy Rate  Crude Death Rate  Child Mortality Rate  \
0           NaN            43.019718         19.865000                 228.6   
1           NaN            18.157681         21.720000                 247.8   
2           NaN            43.019718          6.742000                  67.6   
3           NaN                  NaN         11.387000                 124.6   
4           NaN                  NaN         19.176000                 243.2   
..          ...                  ...               ...                   ...   
187         NaN                  NaN         11.900000                  25.6   
188         NaN                  NaN         11.300000                  24.2   
189        35.9                  NaN         11.900000                  11.4   
190         NaN                  NaN          9.984000                  53.3   
191      

## Part 1: What do you want to learn about?

Read this <a href="https://berkeley.box.com/s/00t6q8gwci1en5bqhtk9nd6pg5jkjwk1">codebook</a>, which contains some description of the variables in the data.

1.1 Identify two pairs of variables that may be related to each other.

1.2 For each pair, come up with a theoretical argument about why one might cause the other.

1.3 Given the theoretical argument you provided and how the variables are measured, do you have any concerns about the reliability or validity of these variables? (As we haven't given too much information about how the data were collected, it's ok to just discuss hypothetical issues)

## Part 2: Descriptives

[For this part, you may want to refer back to the week 9 notebook and problem set 3]

2.1 Identify a categorical, ordinal, and numeric variable in the data set. (You may use the variables identified in part 1.1 or others). 

2.2 Write code to come up with descriptive statistics for these three variables. What are typical values? What does the distribution look like? 

2.3 Repeat this process for the remaining variables you identifed in part 1.1

## Part 3: Relationships

[For this part, you may want to refer back to the week 12 and 13 notebooks and problem set 4]

3.1 For each pair of variables from 1.1, come up with an appropriate way to measure the strength of the relationship between the variables. If possible, come up with a visual way to represent this.

3.2 For each pair, conduct a hypothesis test where the null hypothesis is that there is no relationship between the two variables. What do you learn from these tests?

## Part 4: Causal inference

[For this part, you may want to refer back to the week 14 notebook and problem set 4]

4.1 If you found a strong relationship between the variables in part 3, what might be some alternative explanations? If you did not find a strong relationship, could there be confounding variables which mask a true causal relationship? 

4.2 Identify a variable (Z) in the dataset which could be a confounding variable for one of the relationships you studied (ideally corresponding to your answer from 4.1, but it's ok if not). Look at the relationzhip between your main independent and dependent variables and Z. Does this provide evidence that Z is an important confounding variable?

4.3 Run a multivariate regression which includes the variable Z from 4.2 as a control variable. How are the results different than a bivariate regression without Z? 



## Part 5: Next Steps

5.1 Summarize what you learned from parts 1-4.

5.2 If you wanted to learn more about the causal relationships from part 1, what additional data would be useful to collect?

5.3 Think of an alternative research design which would provide better causal evidence for this relationship. Alternatively, could you ask a slightly different question which would allow for a more credible causal analysis?